### PR TITLE
Create basic sensor device

### PR DESCRIPTION
**Pull request by connormuench**
Addresses #5.

Created a temperature sensor that syncs with the coordinator that is on the coordinator branch. Documentation for the device can be found in /devices/DS18B20TemperatureSensor/README.md.

Also established a commit policy and development instructions for devices in /devices/README.md.

### CODE REVIEW

File: devices/DS18B20TemperatureSensor/main.c [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+ */
+int callback(const xbee_cmd_response_t FAR *response)
+{
+	if (response->flags & XBEE_CMD_RESP_FLAG_TIMEOUT || 
+	    response->flags & XBEE_AT_RESP_ERROR || 
+	    response->flags & XBEE_AT_RESP_BAD_PARAMETER || 
+	    response->flags & XBEE_AT_RESP_BAD_COMMAND)
+		at_ready = FALSE;
+	else {
+		sleep_time = (int)(response->value);
```
Comment (wisp4400) [inline devices/DS18B20TemperatureSensor/main.c:80]: why are we doing sleep time when setting it to be ready?

File: devices/DS18B20TemperatureSensor/main.c [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+		tick_xbee();
+	}
+	printf("Coordinator accepted");
+	
+	// Wait 2 seconds to clear out any initialization commands that may still be queued
+	rtc_set_timeout(HZ * 2);
+	while (!rtc_timeout_expired())
+		tick_xbee();
+	
+	for (;;) {
```
Comment (wisp4400) [inline devices/DS18B20TemperatureSensor/main.c:182]: what does this do?  is it a while true?

**Review by wisp4400 — APPROVED** [review]

File: devices/DS18B20TemperatureSensor/main.c (hunk 2 above)
Comment (connormuench) [reply devices/DS18B20TemperatureSensor/main.c:182]: Yes, it's an infinite loop. It's the auto-generated infinite loop by CodeWarrior.

File: devices/DS18B20TemperatureSensor/main.c (hunk 1 above)
Comment (connormuench) [reply devices/DS18B20TemperatureSensor/main.c:80]: at_ready is a variable that indicates a value has been received from an XBee AT parameter. Also, we're just storing the sleep time here, not actually sleeping the device.

**Pull request merged**